src/index.html

```diff
@@ -7,6 +7,403 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+         src="images/logo.png"
+         alt="logo"
+         class="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
```

Comment:
It's great you've added blank spaces between sibling blocks, but you shouldn't add them between parent and child block (Checklist #2)